### ISSUE TITLE
ENOENT: no such file or directory

### ISSUE THREAD

**Issue by ibutbul1** [microsoft/vscode-azureAutomation#33]
Does this occur consistently? Yes

Repro steps:

1. Connect to the Azure Automation Extension
2. Open any runbook from an Automation Account

Action: automation.openLocal
Error type: ENOENT
Error Message: ENOENT: no such file or directory, open 'c:\Program Files\Microsoft VS Code//vscodeAutomation//redacted:id//RUNBOOKNAME

Version: 1.1.0
OS: win32
OS Release: 10.0.22621
Product: Visual Studio Code
Product Version: 1.77.3
Language: en

<details>
<summary>Call Stack</summary>

```
extension.js:2:1676472extension.js:2:1676472
n extension.js:2:1669894
```

</details>

**Comment by Shruti-MS** [comment]
Hi @ibutbul1 , this happens because the current directory which is set to store runbooks i.e "c:\Program Files\Microsoft VS Code//" does not have the permission to create file/directory.
Please change the directory using "Change Directory"(Ctrl+Shift+P) command in command palette. You can change directory to location where it VSCode have permission to write files.

Please refer to this issue:-https://github.com/microsoft/vscode-azureAutomation/issues/15

**Comment by codaamok** [comment]
Thanks for providing a workaround.

I just installed the extension and the first thing I did was try and open an existing runbook. However I got this issue. Perhaps the default directory should be different? This would improve first user experience rather getting an immediate error.

I suspect this happens for people who have system-wide vscode installed.

Program Files is not a suitable directory for these temporary files. Perhaps consider using the temp variable instead.
